### DISCUSSION TITLE
Which Script for TASMOTA Smart Meter Interface with EasyMeter Q3DA 1004 V3.04?

### DISCUSSION THREAD

**Discussion by Infosucher** [arendst/Tasmota#20563] [Support request]
I have an electricity meter from EasyMeter Typ -> Q3DA 1004 V3.04. Which script exactly do I need so that the smart meter gives me the correct values? Do I still need to adapt the script? If so, what exactly needs to be adjusted?

Thank you very much.

**Comment by HofeBY** [comment]
https://tasmota.github.io/docs/Smart-Meter-Interface/#easymeter-q3d-q3da1024-obis
